src/main/java/seedu/contax/logic/parser/ParserUtil.java

```diff
@@ -238,4 +238,23 @@ public static Duration parseDuration(String duration) throws ParseException {
 
         return new Duration(convertedDuration);
     }
+
+    /**
+     * Parses a {@code String userCommand} into a {@code command with index}.
+     *
+     * @throws ParseException if the given {@code command} is empty.
```

Comment:
```suggestion
     * @throws ParseException if the given {@code userCommand} is empty.
```
maintain the same naming

Reply:
thx changed